WSEvents.h

```diff
@@ -41,9 +40,18 @@ class WSEvents : public QObject {
     uint64_t GetRecordingTime();
     const char* GetRecordingTimecode();
 
+    bool Heartbeat_active;
+    bool Heartbeat_CurrentProfile_active;
+    bool Heartbeat_CurrentScene_active;
+    bool Heartbeat_Streaming_active;
+    bool Heartbeat_Recording_active;
+    bool Heartbeat_TotalStreamTime_active;
+    bool Heartbeat_TotalRecordTime_active;
```

Comment:
-First of all thanks a lot for including it. I will run by all you comments.
-Why add/sub: I thought lets keep is small an neat. Also if in future extra key/value pairs are coded than it can grow without everybody needing the data. (me once a programmer of 8-bit micro controllers.)